consensus/consensus.md

```diff
@@ -177,6 +180,20 @@ There is no need to consider Key Blocks' headers as Key Blocks do not hold heavy
 
 #### Micro Blocks
 
+```
+PROTOCOL_VERSION: 21
+GENESIS_VERSION: 21
+```
+
+- the timestamp of a micro block MUST be smaller than `now() + 10m`
+- the timestamp of a micro block MUST be bigger than:
+  - the timestamp of the previous block + 3s (if previous block is a micro block)
```

Comment:
I see in code this "bigger" means "bigger or equal". Change detailing this would be welcome.